src/MagnumPlugins/AssimpImporter/Test/material-raw.gltf

```diff
@@ -17,11 +17,52 @@
     "meshes": [
         {
             "primitives": [{
-                "attributes": {},
+                "attributes": {
+                    "POSITION": 0,
+                    "TEXCOORD_0": 1,
+                    "TEXCOORD_1": 1,
+                    "TEXCOORD_2": 1,
+                    "TEXCOORD_3": 1
+                },
```

Comment:
Maybe add a note why 5.1.0+ needs these attribs? So I don't accidentally remove them later :)

Reply:
I added them because gltf-validator complained 😄 But they're not needed it turns out, I removed them again.